### PR TITLE
NEWTON-610: Parse report contents to generate more descriptive title

### PR DESCRIPTION
**Pull request by victorromeroguardiola**
Uploaded all the part related to the PDFs and the automatic save on output folder.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+
+This is a tool designed to be used as stand alone. The installation should be done by performing a normal npm installation
+
+```bash
+npm install
+```
+
+### Configuration
+
+
```
Comment (joaquimvila) [inline README.md:16]: Need to complete this section

File: package.json [hunk 2]
```diff
@@ -4,21 +4,22 @@
     "description": "Scripts to use AllureReporter tool from a continuous integration environment",
     "main": "index.js",
     "scripts": {
-        "test": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register",
-        "report": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --reporter spec --timeout 999999 --colors --full-trace"
+        "report": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --config .mocharc.json",
+        "report-jenkins": "./node_modules/.bin/mocha -r ts-node/register -r tsconfig-paths/register --spec src/index.ts --config .mocharc-jenkins.json"
```
Comment (joaquimvila) [inline package.json]: We do not need this script as reports are always generated.

**Review by joaquimvila — APPROVED** [review]
Agree with changes

File: README.md (hunk 1 above)
Comment (victorromeroguardiola) [reply README.md:16]: ok as soon as possible

File: package.json (hunk 2 above)
Comment (victorromeroguardiola) [reply package.json]: Done

**Pull request merged**